### DISCUSSION TITLE
Why window scrolls to top always on router.back()?

### DISCUSSION THREAD

**Discussion by svobik7** [vercel/next.js#20864] [Help]
After upgrading to nextjs 10.0.5 today I noticed that window is always scrolled to top when router.back() is used. Is this new behavior or some kind of bug?

I am showing post details in dialog abouve post list using router.push() and closing using router.back(). So every time user closes dialog he lost his scroll position in post list. It is very anoying.

**Comment by leerob** [comment]
Yes, `next/router` is now consistent with `next/link`. However, `next/link` allows you to [disable this](https://nextjs.org/docs/api-reference/next/link#disable-scrolling-to-the-top-of-the-page). Might be worth adding the same option for `next/router`.
